packages/components/Atoms/Icon/TextIcon.vue

```diff
@@ -86,5 +102,21 @@ export default {
     &--after {
       margin: $rpl-text-link-after-margin;
     }
+
+    &__url-icon {
+      display: flex;
+    }
+
+    &__absolute-url {
+      display: block;
```

Comment:
This will bring down the whole text to the next line. Try inline-block?

Reply:
@rubyllamadora @mayurngondhkar Not required, but I reckon this is a good candidate for a mixin - See Chris Coyiers https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/#for-the-scss-inclined

`@mixin rpl_word_wrap`